ansible/playbook/tasks/update.yml

```diff
@@ -50,6 +50,14 @@
       include_role:
         name: grafana
 
+    - name: Enable cron autostart in non-docker distros
+      when: not is_docker
+      ini_file:
+        path: /etc/supervisord.d/pmm.ini
+        section: program:cron
+        option: autostart
+        value: "true"
```

Comment:
I think we have regular cron on AMI/OVA images. We don't need to start it from a supervisor.